vendors/Makefile

```diff
@@ -0,0 +1,14 @@
+# Maybe this should be a bash script.
+# This is however only ment to be run by developers.
+
+help:
+	@echo "make one of the following targets" 
+	@echo
+	@egrep "^[0-9a-zA-Z\-]+:$$" Makefile | grep -v "help:"
+
+update-preexec:
+	# - Maybe we should make this into a git-subtree(1) instead
+	# - Maybe we should always supply a bash-it-init.bash file to the vendoer forlder so we have a 
```

Comment:
I also agree with this comment, this will make the loading way more clear and rebust